cli/internal/util.go

```diff
@@ -108,7 +108,7 @@ func ParseKeyPair(p []string) map[string]string {
 	params := map[string]string{}
 	for _, i := range p {
 		parts := strings.SplitN(i, "=", 2)
-		if len(parts) != 2 {
+		if len(parts) != 2 { //nolint:gomnd
```

Comment:
all strings.SplitN(..., 2) should be refactored to use strings.Cut

Reply:
Done